tests/vxlan/test_vxlan_fields.py

```diff
@@ -0,0 +1,133 @@
+def test_vxlan_fields(api, b2b_raw_config_vports, utils, tx_vport, rx_vport):
+    """
+    Configure three raw vxlan flows with ,
+    - fixed pattern for all vxlan fields
+    - list pattern for all vxlan fields
+    - counter pattern for all vxlan fields
+    Validate,
+    - fetch the vxlan header via restpy framework and validate
+      against expected.
+    """
+    # fixed
+    outer_src_mac = "00:00:0a:00:00:01"
```

Comment:
We also validate packet through "e2e" test where validate it in capture level. Please check if we can incorporate additional UT with "@pytest.mark.e2e" marker. Please check this test for more details https://github.com/open-traffic-generator/snappi-ixnetwork/blob/main/tests/udp/test_udp_fixed_ports_e2e.py

Reply:
@alakendu okay, I will do it.